src/skin/colorschemeparser.cpp

```diff
@@ -39,6 +39,12 @@ void ColorSchemeParser::setupLegacyColorSchemes(QDomElement docElem,
             }
         }
 
+        if (!bSelectedColorSchemeFound) {
+            // If we didn't find a matching color scheme, pick the first one
+            schemeNode = schemesNode.firstChild();
+            bSelectedColorSchemeFound = true;
```

Comment:
bSelectedColorSchemeFound = !schemeNode.isNull() 

This also resolves the issue discussed below.